pkg/ddevapp/templates.go

```diff
@@ -12,7 +12,8 @@ services:
       - "./data:/db"
     restart: always
     environment:
-      - TCP_PORT=$DDEV_HOSTNAME:{{ .dbport }}
+      # TCP_EXPOSE allows for TCP ports to be accessible from <site>.ddev.local:<port>
```

Comment:
Please add an example (preferably with colon format) to this lovely comment. Thanks.